ocp_resources/machine_config_pool.py

```diff
@@ -2,8 +2,74 @@
 
 
 class MachineConfigPool(Resource):
+    """
+    MachineConfigPool object. API reference:
+    https://docs.openshift.com/container-platform/4.12/rest_api/machine_apis/machineconfigpool-machineconfiguration-openshift-io-v1.html
+
+    Args:
+        machine_config_selector (object): Matching labels/expressions, to determine which MachineConfig objects
```

Comment:
What do you mean by `(object)` ? what should the user send here?

Reply:
The user should send either a `matchExpressions` or a `matchLabels`  object that will be put in the `machineConfigSelector` object, as it is specified in the API. That's why I wrote "Matching labels/expressions".

Reply:
Python object? do have such one?

Reply:
Actually it's a dict.
I changed the type.